ang/crmUi.js

```diff
@@ -642,7 +642,7 @@
             $timeout(function () {
               // ex: msg_template_id adds new item then selects it; use $timeout to ensure that
               // new item is added before selection is made
-              element.select2('val', ngModel.$viewValue);
+              element.select2('val', ngModel.$modelValue);
```

Comment:
@colemanw The same issue does affect all multi-value instances of select2 created via angular directive. I would say that it should be changed at L602 as well. I haven't done any testing of how this might affect the crmUiSelect directive or looked for specific use cases and I'm not sure how complete the automated JS tests are. But it would seem to make sense to do the change and run it through the JS tests to see if something breaks.